apps/caption-image/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Pull base image.
+ARG VERSION=latest
+FROM aperturedata/workflows-base:${VERSION}
+
+ENV APP_NAME=workflows-caption-image
+
+COPY requirements.txt /
+RUN pip install -U pip
+RUN pip install torch torchvision --index-url https://download.pytorch.org/whl/cpu
+RUN pip install --no-cache-dir -r /requirements.txt
+
+COPY app/weights.py /app/weights.py
```

Comment:
I think this is intended for cache warmup. It might be better to make that explicit in the name.

Reply:
sounds good. will cahnge.